spec/amazon_order/parsers/order_spec.rb

```diff
@@ -53,6 +40,19 @@ def ensure_fixture_filepath(path)
               expect(parser.orders.to_json).to match(/\[{"order_placed":/)
             end
           end
+
+          describe '#shipments' do
+            it 'has shipments' do
+              expect(order.shipments.size).to be > 0
+            end
+          end
+
+          describe '#products' do
+            it 'has products' do
+              expect(order.products.size).to be > 0
```

Comment:
I expect these 2 examples (`#shipments, #products`) run only with 'spec/fixtures/files/order-2018-p1-20180129141532-contains-home-service.html' because some pages don't have shipments.

I'm checking arbitrary pages with examples in `Dir.glob("#{TARGET_DIR}/*html")` block.